core/src/ast_builder/expr/function.rs

```diff
@@ -132,6 +134,18 @@ impl TryFrom<FunctionNode> for Function {
             FunctionNode::Concat(expr_list) => expr_list.try_into().map(Function::Concat),
             FunctionNode::Degrees(expr) => expr.try_into().map(Function::Degrees),
             FunctionNode::Radians(expr) => expr.try_into().map(Function::Radians),
+            FunctionNode::Ltrim(expr_node, chars_node) => {
+                let chars = chars_node.map(|chars| chars.try_into().unwrap());
```

Comment:
```suggestion
                let chars = chars_node.map(TryInto::try_into).transpose()?;;
```

<https://doc.rust-lang.org/stable/core/option/enum.Option.html#method.transpose>

Reply:
Thank you, I will apply the following changes as guided.